contracts/governance/Staking/Staking.sol

```diff
@@ -373,6 +461,58 @@ contract Staking is
         emit StakingWithdrawn(msg.sender, amount, until, receiver, isGovernance);
     }
 
+    /**
+     * @notice Send user' staked tokens to a receiver.
+     * This function is dedicated only for direct withdrawal from staking contract.
+     * Currently only being used by cancelTeamVesting()
+     *
+     * @param amount The number of tokens to withdraw.
+     * @param until The date until which the tokens were staked.
+     * @param receiver The receiver of the tokens. If not specified, send to the msg.sender.
+     * @param vestingConfig The vesting config.
+     * @dev VestingConfig struct intended to avoid stack too deep issue, and it contains this properties:
+        address vestingAddress; // vesting contract address
+        uint256 startDate; //start date of vesting
+        uint256 endDate; // end date of vesting
+        uint256 cliff; // after this time period the tokens begin to unlock
+        uint256 duration; // after this period all the tokens will be unlocked
+        address tokenOwner; // owner of the vested tokens
+     * */
+    function _withdrawFromTeamVesting(
+        uint96 amount,
+        uint256 until,
+        address receiver,
+        VestingConfig memory vestingConfig
+    ) internal {
+        address vesting = vestingConfig.vestingAddress;
+        /// require the caller only for team vesting contract.
+        require(vestingRegistryLogic.isTeamVesting(vesting), "Only team vesting allowed");
+
+        // @dev it's very unlikely some one will have 1/10**18 SOV staked in Vesting contract
+        if (amount == 1) {
+            return;
+        }
```

Comment:
we don't need this check here
@ororopickpocket can you pls confirm

Reply:
that's right. it was only required because of the _withdraw, _withdrawNext workaround.

Reply:
fixed in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/21dd284dc55c0b0355bb165ce2e24dbf154e32e6)